.github/workflows/maintenance-tagging-workflow.yml

```diff
@@ -0,0 +1,26 @@
+name: Add or remove maintenance labels
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  add_label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: "Get @breaking-change annotation counts"
+      id: deprecations
+      run: echo "::set-output name=count::$(git diff ${{ github.event.pull_request.base.sha }} ${{ github.sha }} | egrep "^\+" | grep '@breaking-change' -c)"
```

Comment:
I thought we only want to check for commit messages, if there is written "BREAKING CHANGE:" ? @kyubisation What's your opinion?

Reply:
Yes, just checking the PR message should work best.
To explain: We sometimes add a note `@breaking-change 15` to indicate that something should be changed for version 15, which will be a breaking change, but is not with the current change.